eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -147,6 +147,17 @@ public TjSmart foreign() {
         );
     }
 
+    /**
+     * Tojo for eo-external.* file.
+     *
+     * @return TjSmart of the current eo-external.file.
+     */
+    public TjSmart external() {
```

Comment:
@maxonfjvipon I'm not sure here, but can we left `package-private` modifier for that method?

Reply:
@volodya-lombrozo why do you think so? There's a place (and will be more) outside the class where the method is called. The same story as with `foreign()` method

Reply:
@maxonfjvipon I've found the only place where we call this method outside of the class:
https://github.com/objectionary/eo/pull/2219/files#diff-7e783d4f9fc150a8ce469c472b3d3e567c440f288fcf0fd9f6bd531ddfefcf6aR171

And this place is inside of the same package, so we can leave just `TjSmart external()` without `public` modifier